change/@azure-msal-browser-1fe30015-5db8-41ae-988f-d27fa55e15e3.json

```diff
@@ -0,0 +1,7 @@
+{
+  "type": "prerelease",
+  "comment": "Export IdTokenClaims & PromptValue types",
+  "packageName": "@azure/msal-browser",
+  "email": "thomas.norling@microsoft.com",
+  "dependentChangeType": "patch"
```

Comment:
AFAIK `dependentChangeType` should be `prerelease` as well until we GA.

Reply:
We've never bothered with this field, AFAIK it only impacts where the dependency version bump log appears in the changelog but does not affect the bump itself.